.gitignore

```diff
@@ -264,3 +264,5 @@ BenchmarkDotNet.Artifacts/
 BDN.Generated/
 binaries/
 
+# Do not exclude any external files
+!external/**/*
```

Comment:
Yes, it is required and will be required each time we update the binaries. I intentionally put the binaries in an external folder so that the exclusion was contained.